site/app/controllers/EmailStatusController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace app\controllers;
+
+use app\libraries\Core;
+use app\libraries\response\MultiResponse;
+use app\libraries\response\WebResponse;
+use app\libraries\response\JsonResponse;
+use app\libraries\response\RedirectResponse;
+use app\models\email\EmailStatusPage;
+use Symfony\Component\Routing\Annotation\Route;
+use app\libraries\routers\AccessControl;
+
+class EmailStatusController extends AbstractController {
```

Comment:
As this page is only meant to be viewable by instructors, we should add an access control on this class to ensure no other classes can access it via direct URL.

```suggestion
/**
 * @AccessControl(role="INSTRUCTOR")
*/
class EmailStatusController extends AbstractController {
```